templates/configs/base.conf.j2

```diff
@@ -174,11 +175,11 @@ export BM_TARBALL_EXTRA_OPTIONS=""
 
 # Which frequency to use for the master tarball?
 # possible values: weekly, monthly
-export BM_TARBALLINC_MASTERDATETYPE="weekly"
+{{ macros.config_row(config, 'BM_TARBALLINC_MASTERDATETYPE', 'weekly') }}
 
 # Number of the day, in the BM_TARBALLINC_MASTERDATETYPE frequency
 # when master tarballs should be made
-export BM_TARBALLINC_MASTERDATEVALUE="1"
+{{ macros.config_row(config, 'BM_TARBALLINC_MASTERDATEVALUE', '1') }}
```

Comment:
You can pass directly 1 (as integer) and not '1' (string) as the macro always double quotes values, even in the cases of integer or boolean :)